src/tools/decomp.py

```diff
@@ -53,135 +100,388 @@
 
 import math
 import time
+from loadbal import qbicity, changeIndex, halfDecomp, addHsymmetry, adaptNeiList, reDistCellsHom, nodeGridSizeCheck
+
+__author__ = 'Horacio V Guzman'
+__email__ = 'horacio.v.g at gmail dot com'
+__version__ = '1.0'
+__all__ = [
+    'nodeGrid', 'cellGrid', 'cherrypickTotalProcs',
+    'neiListHom', 'neiListAdress'
+]
+
+
+# WARNING! New arguments are needed! At least...box_size,rc,skin
+
+
+def nodeGrid(box_size, rc, skin, n, eh_size=0, ratioMS=0, idealGas=0, slabMSDims=[0, 0, 0]):
```

Comment:
Or perhaps:
```py
import numbers

def nodeGrid(box_size=None, rc=None, skin=None, n=None, eh_size=0, ratioMS=0, idealGas=0, slabMSDims=[0, 0, 0,]):
    is isinstance(box_size, numbers.Number) or (box_size is None and n is not None):
        return nodeGrid_OLD(box_size)
```

Reply:
Good Idea, this will complete the warning I already included